etl/steps/data/garden/insee/2024-04-26/relative_poverty_france.meta.yml

```diff
@@ -0,0 +1,49 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    display:
+      name: Share in relative poverty
+      numDecimalPlaces: 1
+    processing_level: major
+    description_key:
+      - Income is ‘post-tax’ — measured after taxes have been paid and most government benefits have been received.
+      - Income has been equivalized – adjusted to account for the fact that people in the same household can share costs like rent and heating.
+    presentation:
+      title_public: Share in relative poverty
+      topic_tags:
+        - Poverty
+        - Economic Inequality
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  relative_poverty_france:
+    variables:
+      headcount_ratio_40_median:
+        title: 40% of the median - Share of population in poverty
```

Comment:
minor comment: I generally, and personally, prefer to use `,` instead of a hyphen `-` to split indicator variables. The reason is that when there are dimensions, these are appended to the title using a hyphen `-`.

Reply:
Thanks. So it would work the other way around if it was dimensional. I will have it in mind